virtual_workstation_setup.sh

```diff
@@ -12,6 +12,27 @@ silent_grep() {
     command grep -q  > /dev/null 2>&1 "$@"
 }
 
+set +e
+for filename in ~/.ssh/id_*; do
```

Comment:
Let's preemptively add a comment saying something like: If the user has a different name for the ssh key, we assume they know about the agent and can set it up manually. This check is for the majority of users.

Reply:
done